packages/eslint-config-angular/i18n-attributes-ignore.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+    nimble: [
+        'action-menu-slot',
+        'appearance',
+        'appearance-variant',
+        'column-id',
+        'field-name',
+        'format',
+        'icon',
+        'id-field-name',
+        'key',
+        'key-type',
+        'orientation',
+        'severity',
+        'slot',
+        'theme'
+    ],
+    'systemlink-lib-angular': [
+        // sl-workspace-selector
+        'action',
+
+        // sl-table
+        'columnId',
+        'fieldName',
+        'idFieldName',
+        'selectionMode',
+        'slTableColumnId',
+
+        // sl-grid
+        'dataRowId',
+        'loadColumnStateBehavior',
+        'parentDataField'
+    ],
+    jqx: [
+        // smart-table
+        'sortMode',
+
+        // smart-query-builder
+        'applyMode',
+        'fieldsMode'
+    ],
+    material: [
+        'cdkDragPreviewContainer',
+        'floatLabel',
+        'fontIcon',
+        'fontSet',
+        'matColumnDef',
+        'matTooltipClass',
+    ]
```

Comment:
Are we loosing the DEFAULT_ALLOWED_ATTRIBUTES ?

https://github.com/angular-eslint/angular-eslint/blob/20bbfb20ef0a0e4d76b7482dc8263492c819e9f2/packages/eslint-plugin-template/src/rules/i18n.ts#L112C3-L112C19

If so how should we manage them, doesn't look like they are exported.. copy and paste in with link back?

Reply:
No, we aren't losing them. Lower in that file you can see they [concat the two together](https://github.com/angular-eslint/angular-eslint/blob/20bbfb20ef0a0e4d76b7482dc8263492c819e9f2/packages/eslint-plugin-template/src/rules/i18n.ts#L219). And you can see in the test I added the i18n tag is still needed for a `title` attribute.

Reply:
Scanning that source it does look like we could scope these to specific elements: https://github.com/angular-eslint/angular-eslint/blob/20bbfb20ef0a0e4d76b7482dc8263492c819e9f2/packages/eslint-plugin-template/src/rules/i18n.ts#L525

Not sure that is super valuable but it is an option. Might be useful to have a comment on the top of the list describing the recommendation and stating element specific attributes are possible

Reply:
I've update the attribute list to include a block comment that describes the list's purpose, the intent that should be followed when adding to it, and mentions to option to use `element[attribute]` notation for more specific scoping.